digitalpublicgoods/notesmaster.json

```diff
@@ -0,0 +1,119 @@
+{
+  "name": "NotesMaster",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://notesmaster.com/termsofuse"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "System requirements - https://notesmaster.com/systemrequirements"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "No",
+    "checkNonPIIAccessMechanism": "",
+    "nonPIIAccessMechanism": ""
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "Federal Children’s Online Privacy Protection Act of 1998",
+      "U.S. Family Educational Rights and Privacy Act"
+    ],
+    "adherenceSteps": [
+      "https://notesmaster.com/privacy"
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "W3C Accessibility Standards",
+      "WCAG 2.1 standards",
+      "Creative Commons standards"
```

Comment:
As far as I know, there is no such a thing as "Creative Commons Standards". Creative Commons is a set of *standard* licenses, but there is no standard to comply with. Instead, one uses those licenses to release their content. I suggest to further clarify or remove from this list.

Reply:
Removed in commit 7eda5ff